CHAPI Multiple Credential Share failure

Wallet fails with following error when trying to share multiple credential through CHAPI 

> runtime-core.esm-bundler.js:226 Error: agent wasm: code: 12021, message: failed to read credential manifest: invalid credential manifest: the ID default_tbwallet_output appears in multiple output descriptors
>     at agent.js:1:1560

The latest wallet code deployed in dev fails with [this](https://github.com/trustbloc/sandbox/blob/main/docs/demo/new-bank-account-usecase.md) usecase.

Possible issue:
The output descriptors [here](https://github.com/trustbloc/wallet/blob/4b2d9826aaf36a57c1174f025a499a2559bd8ec9/cmd/wallet-web/src/mixins/common/helper.js#L335) contain duplicate ids when multiple credentials don't have descriptors defined. We need to see if removing duplicated would fix the issue.

--- a/cmd/wallet-web/src/mixins/common/helper.js
+++ b/cmd/wallet-web/src/mixins/common/helper.js
@@ -301,6 +301,7 @@ export function prepareCredentialManifest(presentation, manifestDictionary, issu
     version: '0.1.0',
     output_descriptors: [],
   };
+  let uniqueOutputDescriptors = new Set();
 
   let fulfillment = {
     id: uuidv4(),
@@ -315,7 +316,7 @@ export function prepareCredentialManifest(presentation, manifestDictionary, issu
       credential
     );
     if (entry) {
-      credentialManifest.output_descriptors.push(...entry['output_descriptors']);
+      uniqueOutputDescriptors.add(...entry['output_descriptors']);
     } else {
       // find default output descriptor
       entry = _findOutputDescriptor(manifestDictionary['VerifiableCredential'], credential);
@@ -332,7 +333,7 @@ export function prepareCredentialManifest(presentation, manifestDictionary, issu
           });
         }
 
-        credentialManifest.output_descriptors.push(...entry['output_descriptors']);
+        uniqueOutputDescriptors.add(...entry['output_descriptors']);
       } else {
         console.error(
           "couldn't find default credential manifest for given credential",
@@ -353,5 +354,6 @@ export function prepareCredentialManifest(presentation, manifestDictionary, issu
 
   presentation['credential_fulfillment'] = fulfillment;
 
+  credentialManifest.output_descriptors = Array.from(uniqueOutputDescriptors);
   return credentialManifest;
 }

